lib/bigquery/job.js

```diff
@@ -51,11 +58,18 @@ var is = require('is');
  * @constructor
  */
 function Job(bigQuery, id) {
+  ServiceObject.call(this, {
+    parent: bigQuery,
+    baseUrl: '/jobs',
+    id: id,
+    inherit: []
```

Comment:
Could we name this something else, like `borrows`? When I initially saw this I thought maybe you passed in an array of constructors and I got confused.

It also might be worth noting that I saw a reference to `exclude` but not `inherit` within the `ServiceObject` constructor.